language/polkavm/move-to-polka/src/stackless/module_context.rs

```diff
@@ -405,7 +405,9 @@ impl<'mm: 'up, 'up> ModuleContext<'mm, 'up> {
             curr_fn_env.llvm_symbol_name(&curr_type_vec)
         };
 
+        debug!("Checking if {fn_name} exists in current module");
         if self.fn_decls.contains_key(&fn_name) {
+            debug!("Exists. Skipping");
```

Comment:
Tiny Nit: Maybe include the skipped fn_name in the debug message for completeness.